tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java

```diff
@@ -707,6 +718,46 @@ private static boolean shouldIgnoreUri(String uri) {
     return ImmutableMultimap.copyOf(builder);
   }
 
+  // Combines all script tags into one and append it at the bottom of the
+  // document.
+  private static void combineScriptTags(Document document) {
+    Elements scripts = document.getElementsByTag("script");
+    final StringBuilder sources = new StringBuilder();
+    for (Element script : scripts) {
+      if (!script.attr("src").equals("")) continue;
+      sources.append(script.html()).append("\n");
+      script.remove();
+    }
+    // TensorBoard HTMLs tend to be extremely malformed and it can have multiple
```

Comment:
If this is actually JSoup parsing creating the body elements, maybe
amend the comment?